examples/lusid/orders/order-blocks/create-custom-property-on-order-block/2-call-block.sql

```diff
@@ -0,0 +1,33 @@
+/*
+
+-----------
+call Block
+-----------
+
+Description:
+
+    - In this query, we will call the Blocks that i just set
+
+*/
+
+
+-- Loading in block data from Excel spreadsheet
```

Comment:
Don't think it should re-read the spreadsheet

Reply:
Did this so that the spreadsheet can be changed and it still works. But can hardcode the values in there still if that is desirable?

Reply:
removed the spreadsheet read